xarray/tests/test_plot.py

```diff
@@ -1962,7 +1962,7 @@ def test_datetime_hue(self, hue_style):
         ds2.plot.scatter(x="A", y="B", hue="hue", hue_style=hue_style)
 
     def test_facetgrid_hue_style(self):
-        # Can't move this to pytest.mark.parametrize because py35-bare-minimum
+        # Can't move this to pytest.mark.parametrize because py36-bare-minimum
         # doesn't have mpl.
```

Comment:
Unsure if this is comment is still accurate?

Reply:
It didn't work becasue you can't stick `mpl.collections.PathCollection` in a parametrize statement without importing matplotlib. IIRC adding `@requires_matplotlib` didn't help.

So if the bare-minimum environment doesn't have matplotlib, the comment is still likely valid.

Reply:
Ah I understand now, thanks